docs/components/action.md

```diff
@@ -0,0 +1,25 @@
+# Action
```

Comment:
Maybe add all examples from original.. with the variant, etc.

Reply:
Cool will add